subgraph/queries/harvests.py

```diff
@@ -7,7 +11,31 @@
 console = Console()
 
 
-def fetch_tree_distributions(start_timestamp, end_timestamp, chain):
+def _populate_end_of_previous_harvest(tree_distributions: List[Dict]):
+    """
+    This function groups distributions by strategy and adds
+    param end_of_previous_dist_timestamp to each distribution item
+    """
+    grouped_distributions_by_strategy = defaultdict(list)
+    for distribution in tree_distributions:
+        grouped_distributions_by_strategy[distribution['strategy']].append(distribution)
+    for strategy, dists in grouped_distributions_by_strategy.items():
+        grouped_distributions_by_strategy[strategy] = sorted(dists, key=lambda d: d["timestamp"])
+    # For each distribution populate end of the distribution behind it by adding
+    # end_of_previous_dist data point. If this is a first distribution returned from subgraph,
+    # end_of_previous_dist should be same as start of current distribution
+    for __, distributions in grouped_distributions_by_strategy.items():
+        for dist in distributions:
+            if distributions.index(dist) == 0:
+                dist['end_of_previous_dist_timestamp'] = dist['timestamp']
+                continue
```

Comment:
Curiosity is if `continue` is more efficient than just having this be conditional `if` and `else` considering there isn't anything else in this loop.

Reply:
Yeah, well, think it's more matter of taste here. I was thinking that `continue` is more clear semantically here. Like when I am reading code I tend to skip `continue` in loops. `If-else` condition will force you to always go through the condition. That's how I see it